facets_atlasmaker/montage.py

```diff
@@ -0,0 +1,191 @@
+"""Utilities to combine converted images into atlas and to create manifest."""
+import math
+import os
+from absl import logging
+from PIL import Image
+
+
+# Manifest Key names
+MANIFEST_IMAGE_NAME_KEY = 'image_name'
+MANIFEST_SOURCE_IMAGE_KEY = 'source image'
```

Comment:
Nit: `'source image'` should be `'source_image'` (with an underscore) to match the other key names.

Reply:
Updated